lib/quby/answers/services/updates_answers.rb

```diff
@@ -49,6 +46,20 @@ def on_failure(&block)
 
         protected
 
+        def save_raw_params_on_error(new_attributes, &_block)
```

Comment:
Really? Je vind dat een functie niet hoeft aan te geven wat hij doet? 
Dat de functie `gooi_water_op_tim_als_hij_in_brand_staat` jou stiekem ook slaat als je niet in brand staat zou je niet vervelend vinden? Ook al gingen wij die overal toevoegen onder het motto better safe than sorry?

Ik vind &block wel mooi. Geeft aan dat de functie een block zal yielden. Al kan dat natuurlijk mooier en beter met yarddoc.